### ISSUE TITLE
Discord with BetterDiscord works fine with MicaForEveryone but the Background Type needs to be toggled upon every restart

### ISSUE THREAD

**Issue by Skyyblaze** [MicaForEveryone/MicaForEveryone#339]
[X] I've read [FAQ](https://github.com/MicaForEveryone/MicaForEveryone/wiki/FAQ).
[X] I've restarted the affected app after adding rule for it.
[X] I've checked other issues and this is not a duplicate.

**Describe the bug**
Acrylic with Blur Behind doesn't get correctly applied behind Discord upon the first start or restart of the application and is very opaque. I have to manually set the background type to anything else and then back to Acrylic for it to work correctly and be transparent. 

**Rule**
Titlebar Color: Default
Background Type: Acrylic
Corner Mode: Rounded
Extend Frame Into Client Window: Off
Blur Behind: On

**Target Application**
- Discord with BetterDiscord
- Discord version: 1.0.9016 (35236) BetterDiscord version: 217436

**To Reproduce**

1: Setup Discord with BetterDiscord and any transparent theme
2: Enable "Enable Transparency" in BetterDiscord
3: Set MicaForEveryone to Blur Behind Discord
4: Set the Background Type to Acrylic
5: Set Corners to Rounded
6: Restart Discord
7: Acrylic appears very opaque unless you toggle between any other Background Type and then back to Acrylic in MicaForEveryone

**Expected behavior**
Acrylic behind Discord should get rendered correctly without having to toggle between Background Types in Mica for everyone.

**Screenshots**
Acrylic how it appears upon (re)starting Discord:

![grafik](https://github.com/MicaForEveryone/MicaForEveryone/assets/22203339/eed82d01-418a-4b44-a993-fc701285defc)

Acrylic how it appears upon toggling the Background Type to something else and then back to Acrylic:

![grafik](https://github.com/MicaForEveryone/MicaForEveryone/assets/22203339/fae80c3c-3e68-445a-8122-3d5da93a19d2)

**Windows Version**
Windows 11 22H2  Build 22621.1992

**Other tools being used**
[ ] Rectify11
[X] StartIsBack
[ ] Chrome/Firefox themes or config to add system backdrop
[X] Add it yourself (Any other tool that you may use) BetterDiscord

**Additional context**
I know Discord is technically not supported but I'm setting Discord to be completely transparent with a transparent theme and just use the Blur Behind feature of MFE and it works, it just has this annoying needed step to toggle Background Types whenever the PC (or Discord) is restarted.

**Comment by dongle-the-gadget** [comment]
Are you on MFE 1.3.1.1

**Comment by Skyyblaze** [comment]
> Are you on MFE 1.3.1.1

@dongle-the-gadget No, I'm on 1.3.1.2, I actually think it worked properly a few days ago before I updated MFE now that you say it, should I downgrade to test?

**Comment by dongle-the-gadget** [comment]
I currently have Vencord installed, maybe I'll boot up Windows Sandbox and check. What version did you use before this occured?

**Comment by Skyyblaze** [comment]
> I currently have Vencord installed, maybe I'll boot up Windows Sandbox and check. What version did you use before this occurred?

Alright! Looking at the dates it must have either been 1.3.0.0 or 1.3.1.1.

**Comment by dongle-the-gadget** [comment]
I'm thinking 1.3.0.0 is more likely, since 1.3.1.1 has a bug where it would not apply the rules on any new window.

**Comment by Skyyblaze** [comment]
> I'm thinking 1.3.0.0 is more likely, since 1.3.1.1 has a bug where it would not apply the rules on any new window.

Okay well I downgraded to 1.3.0.0 (by copying it over 1.3.1,2) and the problem persisted, I hope someone can look into it. If I can help anyone with debugging please let me know, I happily would do so.

**Comment by Undisputed00x** [comment]
use Backdrop type: None , enable Extended Frame and Blur Behind maybe that helps, also check the theme's css file as there are the theme's customizable properties

**Comment by Skyyblaze** [comment]
@Undisputed00x thanks for the tip I'll try it should the problem happen again. At first I noticed that simply resizing the Discord window by one pixel fixes it but now ever since I use another tweak called Cute Borders the issue never happened again.

I don't know how that works but aslong as it works I don't mind :P

That's the tweak in question: https://github.com/keifufu/cute-borders Perhaps drawing a border around Discord forces MicaForEveryone to initialize correctly for that window.
